### ISSUE TITLE
It exists.

### ISSUE THREAD

**Issue by Ashiix** [shannpersand/comic-shanns#15]
That's it. That's the issue.

**Comment by ThomasRettig** [comment]
It isn't _that_ bad. It just needs more development. That's it. That's the issue.

**Comment by capezotte** [comment]
What do you mean? For us people who have Comic Sans as their UI font, this is the perfect match.

I'm guessing you're one of those people who think my all-pink Windows 95 theme is tacky.

**Comment by ThomasRettig** [comment]
Exactly. Creating a font is hard enough. To @Ashiix — whether or not you dislike it doesn't mean you should devalue people's work.

PS — if you're an anti-freefont curmudgeon, then check out Comic Code by Toshi Omagari (of Tabular Type Foundry). Maybe that'll catch your fancy.

**Comment by Ashiix** [comment]
Haven't checked my issues in a while, apologies for necro. Just messing around, it's a genuinely well made font.

**Comment by ThomasRettig** [comment]
Ah, so sorry if I misunderstood your initial comment. No worries :)

**Comment by linuxer0x42** [comment]
It's holy, truly a work of art.
